modules/tensor_mechanics/include/utils/abaqus/omi_for_c.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
```

Comment:
A comment explaining what this is for would be helpful

Reply:
will add